### PR TITLE
add API URL in README.md

### PR DESCRIPTION
**Pull request by PGhzhang**

### CODE REVIEW

**Comment by rchatterjee** [comment]
You have some unnecessary changes, e.g., in the grade_a2.py and at the last line of README.md. Can you remove those? Thanks.

File: assignments/a2/grade_a2.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def test_tag_association():
     # Post three receipts
     rids = [int(post_receipts()) for i in range(3)]
     test_get_receipts()
-
+
```
Comment (amfleming) [inline assignments/a2/grade_a2.py:91]: Why?

**Review by amfleming — CHANGES_REQUESTED** [review]
Please only change your own line and not the `autograder`

**Comment by PGhzhang** [comment]
Hi I fixed the changes, can you check the pull request again? thanks

**Pull request closed without merging**